src/game/stage/intro.c

```diff
@@ -547,13 +547,26 @@ static void Task_IntroControllerMain(void)
         p->anim = characterAnimsGettingReady[gSelectedCharacter].anim;
         p->variant = characterAnimsGettingReady[gSelectedCharacter].variant;
         p->unk6C = TRUE;
-        p->spriteInfoBody->s.frameFlags |= MOVESTATE_40000;
-        p->spriteInfoLimbs->s.frameFlags |= MOVESTATE_40000;
 
-        if (IS_MULTI_PLAYER) {
-            p->spriteInfoBody->s.palId = SIO_MULTI_CNT->id;
-        } else {
-            p->spriteInfoBody->s.palId = 0;
+        // These are null when super sonic
+#ifdef BUG_FIX
```

Comment:
Just a nit but these If's don't need to be `{ ... }` cos they only need to care about the next line. Then we don't have to mess with the indentation

Reply:
Uhh, yes they do?
I'd rather encapsulate the whole block with one `if` than add 1-2 additional `if`s at the end in the IS_MULTI_PLAYER block.

Reply:
Ohh yes I missed that 🤦